### ISSUE TITLE
Will the training code be public?

### ISSUE THREAD

**Issue by FacePoluke** [Tencent/HunyuanDiT#39]
Great work! I am trying to do some finetuneing tasks on your work and was wondering if you could share your training code? I'm particularly interested in how you calculate loss and choose the scheduler.
I noticed your prediction pipeline outputs sigma, but the DDPM scheduler you're using doesn't seem to have this option. Can you help me solve this question?

Thanks in advance for your help!

**Comment by yestinl** [comment]
This issue appears to be the same as issue https://github.com/Tencent/HunyuanDiT/issues/41. We will close this issue for now. If there are further questions, feel free to reopen it.
